BootloaderCorePkg/Stage1A/Ia32/Vtf0/Ia16/Init16.asm

```diff
@@ -31,6 +31,7 @@ EarlyApInitReal16:
 ; @param[out] ESP   Initial value of the EAX register (BIST: Built-in Self Test)
 ;
 EarlyInit16:
+	cli

```

Comment:
please use white space indentation.

Reply:
In general I agree it is safer to disable interrupt earlier.  
I am just curious if you actually hit a specific issue with interrupt or proposed the change based on code review. 
In IA32  Intel® Architecture Software Developer’s Manual, it defined the CPU state following a reset.  And "IF" interrupt flag in EFLAGS should be always 0.

Reply:
I just proposed the change based on code review as it seems pretty much safe to disable interrupts earlier. Yes, I have also been through IA32 Intel® Architecture Software Developer’s Manual Volume 3A, it says the state of EFLAGS after reset should be 00000002H implies IF=0.

Reply:
I just proposed the change based on code review as it seems pretty much safe to disable interrupts earlier. Yes, I have also been through IA32 Intel® Architecture Software Developer’s Manual Volume 3A, it says the state of EFLAGS after reset should be 00000002H implies IF=0.